docs/source/guides/area_detectors.rst

```diff
@@ -0,0 +1,610 @@
+
+**Basic Configuration:**
+
+.. code-block:: yaml
+
+    # configs/devices.yml - Standard detector configuration
+    my_instrument.devices.ProductionSimDetector:
+    - name: adsim
+      prefix: "IOC:ADSIM:"
+      labels: ["detectors", "primary"]
+
+    # apstools factory configuration
+    apstools.devices.ad_creator:
+    - name: fast_detector
+      # Factory arguments
+      prefix: "IOC:ADSIM2:"
+      detector_class: "SimDetectorCam"  # Use SimDetectorCam for development
+      plugins: ["image", "stats", "hdf5"]
+      labels: ["detectors", "fast"]
+
+.. note::
+   For production, replace ``SimDetectorCam`` with actual detector classes
+   like ``FastCCDDetectorCam``, ``PilatusDetectorCam``, etc.
+
+**Environment-Specific Configuration:**
+
+.. code-block:: yaml
+
+    # configs/devices_aps_only.yml - Production detectors
+    my_instrument.devices.ProductionPilatus:  # Replace with actual detector class
+    - name: pilatus_real
+      prefix: "12IDA:PILATUS:"
+      labels: ["detectors", "primary"]
+      # Custom initialization
+      init_kwargs:
+        acquire_time: 0.1
+        file_path: "/data/pilatus/"
+
+.. code-block:: yaml
+
+    # configs/devices.yml - Development/simulation
+    ophyd.areadetector.SimDetector:
+    - name: adsim_dev
+      prefix: "SIM:ADSIM:"
+      labels: ["detectors", "primary"]
+      # Simulation parameters
+      init_kwargs:
+        noise: true
+        image_width: 1024  # Typical detector dimensions
+        image_height: 1024
+
+Integration with Plans
+~~~~~~~~~~~~~~~~~~~~~
+
+**Detector in Scan Plans:**
+
+.. code-block:: python
+
+    # plans/detector_scans.py - Detector-specific scan plans
+    from bluesky.plans import count, scan
+    from bluesky import plan_stubs as bps
+
+    def detector_count(detector, num=1, delay=0):
+        """Count plan with detector-specific setup."""
+
+        # Configure detector
+        yield from bps.mv(detector.cam.acquire_time, 0.1)
+        yield from bps.mv(detector.cam.num_images, 1)
+
+        # Execute count
+        yield from count([detector], num=num, delay=delay)
+
+    def detector_series(detector, num_images, exposure_time):
+        """Collect a series of images."""
+
+        # Configure for series acquisition
+        yield from bps.mv(detector.cam.acquire_time, exposure_time)
+        yield from bps.mv(detector.cam.num_images, num_images)
+        yield from bps.mv(detector.cam.image_mode, "Multiple")
+
+        # Trigger acquisition
+        yield from bps.trigger_and_read([detector])
+
+**Detector Alignment Plans:**
+
+.. code-block:: python
+
+    # plans/detector_alignment.py - Detector positioning
+    from apstools.plans import lineup2
+    from bluesky import plan_stubs as bps
+
+    def align_detector_distance(detector, distance_motor, nominal_distance):
+        """Align detector to optimal distance."""
+
+        # Scan around nominal position
+        yield from lineup2(
+            [detector.stats1.total],
```

Comment:
Need to add the `detector` object to this plan so that new images will be triggered.  The `detector.stats1.total` signal does not know how to trigger the detector to acquire new images.
```suggestion
            [detector.stats1.total, detector],
```

Since `lineup2` uses the first detector in the list for its analyses, the `detector` object is appended to the list.